base/Cargo.toml

```diff
@@ -7,12 +7,12 @@ edition = "2021"
 
 [dependencies]
 # Our crates
-avail-core = { git = "https://github.com/availproject/avail-core", tag = "node-v1.9.0.0", default-features = false, features = ["runtime"] }
-kate = { git = "https://github.com/availproject/avail-core", tag = "node-v1.9.0.0", default-features = false }
+avail-core.workspace = true
+kate.workspace = true
 
 # Substrate related
 substrate-prometheus-endpoint = {  version = "0.10.0-dev", optional = true }
-sp-std = { git = "https://github.com/paritytech/substrate.git", branch = "polkadot-v1.0.0", default-features = false }
+sp-std = { git = "https://github.com/availproject/polkadot-sdk.git", tag = "polkadot-v1.6.0-patch", default-features = false }
```

Comment:
@Leouarz I think it is the perfect PR to move all `substrate` deps into [Workspace Dependencies](https://doc.rust-lang.org/cargo/reference/workspaces.html#the-dependencies-table).
It would simplify the future upgrades and reduce any mismatch 
Examples:
- Workspace definition for `avail-core` [here](https://github.com/availproject/avail/blob/e654f6614a4e243f2ad330398210734c807ae2d3/Cargo.toml#L16)
- Use workspace dependency [here](https://github.com/availproject/avail/blob/e654f6614a4e243f2ad330398210734c807ae2d3/base/Cargo.toml#L10)